### ISSUE TITLE
Error Deleting Old Canon Pixma MG3620 Driver

### ISSUE THREAD

**Issue by KrowofAgony** [lostindark/DriverStoreExplorer#153]
I am filing this bug report to inform you that I have attempted to delete the Old Canon Pixma MG360 Driver. **It will NOT allow me to do so, and when I clicked the forced removal option, it gives me an error message.** _Please see the error screenshots below (it is in .ZIP file format, cannot upload the screenshots for some strange reason):_

[DriverStore Explorer Canon Pixma MG3620 Error.zip](https://github.com/lostindark/DriverStoreExplorer/files/5683311/DriverStore.Explorer.Canon.Pixma.MG3620.Error.zip)

Please resolve this issue. It is driving me up the wall.

Thank you for your cooperation!

**Comment by JonnyTech** [comment]
Havd you removed the printer from use in control panel and tried booting to safe mode?

**Comment by KrowofAgony** [comment]
Yes I have. As a matter of fact, here is my latest screenshot proving this in Windows 10:

![Driver Store Explorer Canon Printer Issue Safe Mode](https://user-images.githubusercontent.com/7153059/103482906-9b995700-4db1-11eb-9c26-6ab8e19fba47.png)

Sadly, there isn't any "compromise" between Windows Update and this program.

**Comment by AndhikaWB** [comment]
You can check the log file (`Rapr.log`) on executable directory and send it here. You can also `Open Folder Location` (right click on the driver) and delete the folder manually (but then we won't know what caused the error).

**Comment by epman100** [comment]
or try using an older version of driverstore explorer like v0.9.39 to remove the print drivers, KrowOfAgony
OR by using "pnputil.exe -f -d oemxx.inf" from a normal command window (cmd.exe, not from powershell) to remove them

I had to do this on some Win10 systems where I thought I had completely uninstalled the HP printer drivers for my HP envy 4500 and HP envy photo 7800 series printers but there will still some leftover driver files that were not removed
